Classes/CardIOContext.h

```diff
@@ -26,6 +26,7 @@
 @property(nonatomic, assign, readwrite) BOOL disableBlurWhenBackgrounding;
 @property(nonatomic, assign, readwrite) BOOL suppressScanConfirmation;
 @property(nonatomic, assign, readwrite) BOOL suppressScannedCardImage;
+@property(nonatomic, assign, readwrite) CGFloat scannedImageDuration;
```

Comment:
Can you move this new line down below to just before the `allowFreelyRotatingCardGuide` line? That will keep it consistent with the properties order in `CardIOConfig.h`. (Not that we've always been consistent in the past!)